package.json

```diff
@@ -24,11 +24,11 @@
 		"body-parser": "~1.15",
 		"chalk": "~1.1",
 		"compression": "~1.6",
-		"express": "~4.13",
+		"express": "~4.14",
 		"express-hbs": "~1.0",
 		"moment": "~2.13",
-		"pa11y-webservice": "~2.0",
-		"pa11y-webservice-client-node": "~1.2",
+		"pa11y-webservice": "^2.0.1",
+		"pa11y-webservice-client-node": "^1.2.1",
```

Comment:
Well, yes and no. ~2.0 will allow the installation of 2.0.0, which is vulnerable. This is also what tools like [nsp](https://github.com/nodesecurity/nsp) expect.

In practice, you're right, because `npm install` will pick up 2.0.1 over 2.0.0 anyway. I think that it depends if we want to prevent people shooting themselves on the foot, or if we want to make this automated tools happy.